cli/command/image/build_buildkit.go

```diff
@@ -71,6 +73,32 @@ func runBuildBuildKit(dockerCli command.Cli, options buildOptions) error {
 
 	stdoutUsed := false
 
+	if options.squash {
+		// return error rather than print a warning, as squash is often used for hiding secrets
+		return errors.New("squash is not supported by BuildKit. Consider using a multi-stage Dockerfile instead")
```

Comment:
supported but slightly broken
https://github.com/moby/moby/issues/38903

Reply:
So what should we output in error message?

Reply:
We have a PR https://github.com/moby/moby/pull/39187 , probably we can just remove the error here

cc @tonistiigi

Reply:
Any news on this one?

Reply:
ping @tonistiigi ptal